src/pages/sighting/encounters/Encounters.jsx

```diff
@@ -117,7 +117,7 @@ export default function Encounters({
   return (
     <div>
       <ConfirmDelete
-        open={encounterToDelete && !userDataLoading}
+        open={Boolean(encounterToDelete && !userDataLoading)}
```

Comment:
Boolean not required here, right?

Reply:
Actually this fixes a console warning. `null && null` is actually null, not false, so there was a material-ui warning that open was given an invalid type